build.gradle.kts

```diff
@@ -28,8 +28,56 @@ dependencies {
   detektPlugins("io.gitlab.arturbosch.detekt:detekt-formatting:1.23.1")
 }
 
+/**
+ * Dependency forcing for build dependencies; see docs below on this
+ * in the "allprojects" block for explanation why this is necessary.
+ */
+buildscript {
```

Comment:
i read the comment in the allprojects block, but still did not understand why we need this buildscript block with the same force resolution strategy. could you explain here and/or add a bit more context as a comment in the file?

Reply:
@jiyoontbd OK, great - glad you're calling this out. I intended to explain this through "for build dependencies", but maybe that's not coming across? This one isn't about code in the project, but in the build deps themselves; plugins. So the block below doesn't work to resolve it; that's what `buildconfig` handles.

I'll adjust the docs commit to add this extra detail.